### ISSUE TITLE
Announce successful form submissions to screen reader

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#414]
> **Migrated from GitLab**
> **Original Author:** Grigor Malo (`@grgml`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/62

In three steps of the authentication flow - sign up, submit profile and reset password, after the form is submitted successfully, a view tells you the submission was successful, but nothing is announced to screen readers

## Expected behaviour

When form is submitted successfully, the event should be announced to screen readers

## Current behaviour

Form is submitted but nothing is announced to SR

## Steps to reproduce

The behavior can be observed after filling the forms at https://hhmi.cloud68.co/signup, https://hhmi.cloud68.co/signup-profile, or https://hhmi.cloud68.co/password-reset/:token (link is generated and sent via email after completing https://hhmi.cloud68.co/request-password-reset)

1. Access any of the above forms using a screen reader
2. Fill all the required fields any of the above forms.
3. Submit the form
4. You get successful submission screen, but nothing is announced from screen reader

## Environment

All environments

## Possible solution

Wrap the success component with a div[role="alert"]
